patches/chrome-browser-resources-settings-BUILD.gn.patch

```diff
@@ -10,3 +10,11 @@ index e05a2589a3ee6dfaac01e41ed1f5f9df31b6cbe2..19af64fd4cf823a83f0fc0c843821d99
    }
  
    optimize_webui("build_polymer3") {
+@@ -51,6 +52,7 @@ if (optimize_webui) {
+ 
+     deps = [
+       ":unpak",
++      "//brave/browser/resources/settings:unpak",
```

Comment:
can we make //brave/browser/resources/settings:unpak a dep of unpak so we can keep this at one patch?

Reply:
cc: @petemill